source/Hardpoint.cpp

```diff
@@ -218,6 +218,7 @@ void Hardpoint::Fire(Ship &ship, vector<Projectile> &projectiles, vector<Visual>
 	// offset of (.5 * velocity) and that velocity includes the velocity of the
 	// ship that fired them.
 	Point start = ship.Position() + aim.Rotate(point) - .5 * ship.Velocity();
+	Point visualStart = ship.Position() + aim.Rotate(point);
```

Comment:
Notably, `start` later has `aim.Rotate(outfit->HardpointOffset())` added to it. `visualStart` now does not. This means that `visualStart` doesn't only not have the velocity offset that projectiles do, but also isn't rotated by the hardpoint offset anymore.